internal/bminventory/inventory.go

```diff
@@ -3131,6 +3135,11 @@ func logReplyReceived(params installer.PostStepReplyParams, log logrus.FieldLogg
 	}
 }
 
+func shouldBeLogged(params installer.PostStepReplyParams, host *common.Host) bool {
```

Comment:
`shouldBeLogged` could use some documentation and a better name `shouldStepReplyBeLogged`, I think? Why don't we log under those conditions?

Reply:
Maybe put `params.Reply.ExitCode == MediaDisconnected && *host.Status == models.HostStatusError &&
		host.StatusInfo != nil)` inside a var with a name like `isStepReplyDisconnectionError`
		
and create another var `isHostInError := strings.Contains(*host.StatusInfo, mediaDisconnectionMessage)`
		
then the condition can be read more naturally as `return !(isStepRepliDisconnectionError && isHostInError)`

Reply:
I added the doc and changed the method name.